assets/swagger.json

```diff
@@ -4238,6 +4238,19 @@
         "revision": {
           "type": "string"
         },
+        "revisions": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "sourcePositions": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "format": "int64"
```

Comment:
We could probably use int8, I doubt we'll ever have more than 127 source positions to worry about.

Reply:
Unfortunately, the command flagset does not support `int8`. Kept it `int64` to avoid unwanted typecasting between command and api server.